packages/serp-report/src/background/serp-report.js

```diff
@@ -8,7 +8,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
-
+import * as tldts from '../../../tldts/dist/index.esm.min.js';
```

Comment:
It is a pain point, as this code relates on the external `tldts` package. For now, we are including it in dnr extension by copying to the `vendor` path, so this relative path will work. In the next step we should change it to the bare import (which will resolve correctly in both extensions).